### PR TITLE
Follow tech guides

### LINKED ISSUE #4: Follow tech guides
Make sure the gem is following the best practices suggested here https://github.com/rootstrap/tech-guides/tree/master/open-source

Fix README (contributing section)

### PR DESCRIPTION
**Pull request by santib**
Resolves https://github.com/rootstrap/ruby_open_form_validators/issues/4

Adding CodeClimate in another PR since I don't have permissions yet

### CODE REVIEW

**Comment by santib** [comment]
@santiagovidal I'm missing permissions to be able to add Travis and CodeClimate to this repo

**Review by f19ps — APPROVED** [review]
👏

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 
 # rspec failure tracking
 .rspec_status
+Gemfile.lock
```
Comment (juanmanuelramallo) [inline .gitignore]: Why are we ignoring this?

**Review by juanmanuelramallo — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (santib) [reply .gitignore]: because it's a gem

Comment (santib) [reply .gitignore]: https://github.com/rootstrap/tech-guides/blob/master/open-source/developing_gems.md
https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

Comment (santib) [reply .gitignore]: in other words, a gem should work with several versions of gems and ruby, a Gemfile.lock would mean the gem works only with that specific versions

**Review by jpascual1994 — APPROVED** [review]

**Review by fedeagripa — APPROVED** [review]

**Pull request merged**